src/main/java/house/inksoftware/systemtest/domain/config/infra/InfrastructureLauncher.java

```diff
@@ -78,6 +79,8 @@ private void launchDatabase(String type, String image) {
             resourceLauncher = new SystemTestPostgresLauncher(image);
         } else if (type.startsWith("mssql")) {
             resourceLauncher = new SystemTestMsSqlLauncher(image);
+        } else if (type.startsWith("mysql")) {
+            resourceLauncher = new SystemTestMySqlLauncher(image);
```

Comment:
SystemTestMySqlLauncher is is committed?

Reply:
Committed now.